lib/mutant/ast/meta.rb

```diff
@@ -81,7 +44,8 @@ def assignment?
         # @api private
         #
         def attribute_assignment?
-          arguments.one? && attribute_assignment_selector?
+          !Types::METHOD_OPERATORS.include?(selector) &&
+          selector.to_s.end_with?(ATTRIBUTE_ASSIGNMENT_SELECTOR_SUFFIX)
```

Comment:
There are other cases like this. I do not want to mutate into invalid AST, but interpreted as source via unparser, valid source. Even when semantics differ. Such "implicit" mutations should be made explicit.